src/interfaces/breadcrumb-interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface BreadcrumbIF {
+  text: string
+  disabled?: boolean
+  exact?: boolean
+  to?: any
+  href?: string
```

Comment:
Why are these optional? Looks like the template code doesn't handle these undefined properties.

Reply:
Well, a crumb can have either a to or a href, neither are required although 1 is. 
Not sure how to type that.

Reply:
Not easily/cleanly. I see you just pass those into the breacrumb item, so let's ignore this for now.

Are `disabled` and `exact` used atm?

Reply:
Not here but i saw it passed in some of the ui's.

Reply:
If there is no requirement to use disabled and exact then I suggest deleting them.

Yes, this would break UIs that think these do something, but on the other hand those UIs will need a bit of refactoring to use this shared component anyway. I think it would be better to not carry forward properties that do nothing.

Reply:
Ok right on, will do!